.travis.yml

```diff
@@ -1,4 +1,18 @@
 language: node_js
 node_js:
-  - 9
-  - 8
+  - node
+cache: npm
+
+script:
+  - yarn lint
+  - yarn test:coverage
+
+deploy:
+  provider: npm
+  email: dave@apaxsoftware.com
+  api_key: $NPM_AUTH_TOKEN
+  tag: next
+  on:
```

Comment:
Will this cause a release to `npm` anytime you merge in to `master`? This should be a manual process outside of CI.

Reply:
Still not sure about this. Won't this deploy any time you merge in to `master`?